src/core/io/src/4C_io_input_field.hpp

```diff
@@ -316,6 +446,13 @@ namespace Core::IO
         }
         return it->second;
       }
+      if (const PointMapType* map = std::get_if<PointMapType>(&data_))
+      {
+        FOUR_C_THROW(
+            "Accessing a point-based InputField is only possible via the interpolate() function. "
+            "Either provide an element-based field or use the interpolate() function to obtain the "
+            "value.");
```

Comment:
Technically, this is correct. However, the error is instructing the user on two completely different levels of abstraction: either change the input _or_ change the implementation. The fact that this error exists, might indicate that we should separate the InputField types more strongly. We could have something like a cell-wise and an interpolated InputField. Then the types are strong enough to only allow one of the two accessors and remove the runtime error.